src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -120,10 +115,9 @@ const styles = StyleSheet.create({
     left: 0,
     right: 0,
     zIndex: 10,
+    height: '100%',
     paddingTop: 7,
-    paddingBottom: 7,
-    borderBottomWidth: 1,
-    borderBottomColor: COLORS.lightgrey
+    paddingBottom: 7
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height has been set to 100% and the borderBottomWidth and borderBottomColor properties have been removed. Make sure these changes do not cause any layout issues.

```diff
- borderBottomWidth: 1,
- borderBottomColor: COLORS.lightgrey
+ height: '100%',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->